src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/209363380-c4bec5c8-e772-4706-b120-e14a36cbf3ac.png)
Follow the rules for naming [constants](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)